docs/reference/settings/security-settings.asciidoc

```diff
@@ -2320,19 +2321,18 @@ for every request, set to `0` to disable the JWT cache. Defaults to `20m`.
 // tag::jwt-pkc-jwkset-path-tag[]
 `pkc_jwkset_path` {ess-icon}::
 (<<static-cluster-setting,Static>>)
-The file path or URL to a JSON Web Key Set (JWKS) with the public key material that
-the JWT Realm uses for verifying token signatures. If a path is provided,
-then it is resolved relative to the {es} configuration directory. In {ecloud},
-use an absolute path starting with `/app/config/`.
+The file name or URL to a JSON Web Key Set (JWKS) with the public key material that
+the JWT Realm uses for verifying token signatures. A value is considered a file name
+if it does not begin with `https://`. The file name
+is resolved relative to the {es} configuration directory.  Failures to verify a signature
+will result in an attempt to reload the file as this might indicate that the JWT provider
+has rotated the signing keys.
 +
-If a URL is provided, then it must be either a `file` URL or an `https` URL.
+If a URL is provided, then it must begin with `https://` (http:// is not supported).
 {es} automatically caches the retrieved JWK set to avoid unnecessary HTTP
 requests, but will attempt to refresh the JWK upon signature verification
 failure, as this might indicate that the JWT Provider has
 rotated the signing keys.
```

Comment:
Nit: This sentence is somewhat redundant now since we now say almost the same thing 3 lines above?